tests/webtest_tests.py

```diff
@@ -68,10 +67,9 @@ def test_can_register(self):
     def test_sees_error_if_email_is_already_registered(self):
         # A user is already registered
         user = UserFactory(username='foo@bar.com')
-        # Goes to home page
-        res = self.app.get('/').maybe_follow()
-        # Clicks sign in button
-        res = res.click('Create an Account or Sign-In').maybe_follow()
+        # Goes to log in page
+        # @FIXME(hrybacki): No tests written to test landing page sign in
```

Comment:
I believe there are already views tests for the JSON endpoint that the landing page form uses.